lib/negroku/tasks/eye.rake

```diff
@@ -69,6 +83,23 @@ namespace :negroku do
       end
     end
 
+    before "eye:setup", "eye:setup:discovery" do 
+      if was_required? 'capistrano3/unicorn'
+            watch_process(:unicorn, fetch(:unicorn_roles), {
```

Comment:
why did you move this from the `unicorn.rake`? probably is better, I just want to know your thoughts behind the decision.. :)